src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -1050,7 +1062,12 @@
         }
       },
 
-      "required": ["fullyQualifiedLogicalName"]
+      "required": [ "fullyQualifiedLogicalName" ],
+      "dependencies": {
+        "uriBaseId": [ "uri" ],
```

Comment:
If that's what you're asking, I would say "not now". It may be a fine idea, but I propose that we are locked on both the format and the semantics of v1.0.0. We can consider this in a future version.